R/pkg/R/DataFrame.R

```diff
@@ -1268,6 +1268,82 @@ setMethod("dapplyCollect",
             ldf
           })
 
+#' gapply
+#'
+#' Apply a R function to each group of a DataFrame. The group is defined by an input
```

Comment:
update comment:
Group the SparkDataFrame using the specified columns and apply the R function to each group.